Triangulation_2/examples/Triangulation_2/polylines_triangulation.cpp

```diff
@@ -33,20 +28,13 @@ print(const CDTP& cdtp, Cid cid)
 void 
 contexts(const CDTP& cdtp)
 {
-  CDTP::Subconstraint_iterator
-    beg = cdtp.subconstraints_begin(),
-    end = cdtp.subconstraints_end();
-
-  for(; beg!=end; ++beg){
-    Vertex_handle vp = beg->first.first, vq = beg->first.second;
+  for(auto sc : cdtp.subconstraints()){
```

Comment:
`auto` in examples doesn't seem like a good idea to me

Reply:
It is because there is a real bug to fix (even in master)

Reply:
@afabri Which real bug?

Reply:
It derefences to a pair `<Edge, Vertex_list*>` where `Vertex_list*` is not even documented.  And this was already so for master before this PR.

Reply:
Thus the bug is still there, in your branch, no?

`Subconstraint_iterator::value_type` is documented to be `Subconstaint`, that is `std::pair<Vertex_handle, Vertex_handle>`. But it is actually `pair<Edge, Context_list*>`.

Reply:
yes, and as it as was present before, I wanted to address this later in another PR.